plugins/shadow-block-converter/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+      "baseUrl": "./",
+      "outDir": "dist",
+      "declaration": true,
+      "declarationMap": true,
+      "downlevelIteration": true,
+      "experimentalDecorators": true,
```

Comment:
Remove `experimentalDecorators`: I don't think you're using them.

Reply:
Done. Again I did the same thing to the other file.